package/MDAnalysis/topology/PDBParser.py

```diff
@@ -303,6 +313,24 @@ def _parseatoms(self):
         masses = guess_masses(atomtypes)
         attrs.append(Masses(masses, guessed=True))
 
+
+        # Need to pull elements from Atom names
+        substitute_elements = {v.upper():v for k,v in Z2SYMB.items()}
+        if all(elements):
+            elements = guess_types(elements)
```

Comment:
So as far as I can tell, this is still going to cause the same issue of rejecting most valid elements.
The problem here is that passing the elements through to `guess_types` essentially will always cause MDA to make a guess of some form or another (in this case based on FF atom names).

Rather than relying on `guess_types` and attempting to fix its behaviour for this use case (which might have an impact on user code downstream), I would propose here just purely string matching with `Z2SYMB`.

Something like this might work:

```python
periodic_table = set(SYMB2Z) # set over list might not actually be faster here
elements = [ elem.capitalize() for elem in element ]

if all(elem in periodic_table for elem in elements):
    attrs.append(Elements(elements)
else:
    warnings.warn("some warning message here")
```

Reply:
@IAlibay I thought of doing that earlier, but there was another problem in that case. For example, say the elements column has some elements named as Ox, CA+2, S1, POT, N+, etc. In that case, checking based on just string matching would result in wrong elements being added. Hence, I decided to use guess_types first and then do a string matching.
What do you think?

Reply:
So personally I'd consider all those cases to be "non-normal" elements. At least within the scope of this PR, we'd want to just drop them here (maybe we can somehow get a list of all unique "non-normal" elements and feed it to the warning in the `else` case.

In the long term, my view is that guessing is always best applied only to the cases where a "non-normal" element is found, rather than by default guessing for every entry.

Note: here POT and CA+2 would fail because you are reading a two character portion of `line`.

Reply:
Gotcha. I will rewrite this part according to what you said. 
Also, I can add the warning message as requested.
Should I increase the read limits from [76:78] to [76:80]?